src/python/pants/backend/docker/subsystems/dockerfile_wrapper_script.py

```diff
@@ -189,8 +156,11 @@ def copy_source_references(self) -> tuple[str, ...]:
             return tuple(chain(*(cmd.value[:-1] for cmd in self.get_all("COPY"))))
 
     for parsed in map(ParsedDockerfile.from_file, args):
-        if cmd == "putative-targets":
-            for addr in parsed.putative_target_addresses():
+        if cmd == "from-image-addresses":
+            for addr in parsed.from_image_addresses():
+                print(addr)
+        elif cmd == "copy-source-paths":
+            for addr in parsed.copy_source_paths():
```

Comment:
Heh... too little context when looking at the diff in the github UI!

I overlooked there is already a `copy-sources` returned by the `copy_source_references()` method, which does the same thing I believe :P 

Time for some refactoring cleanup here..